Task_functions/index.js

```diff
@@ -1,11 +1,13 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-
-
-
-//Write code for the 2nd task below this
-
```

Comment:
Consider using more descriptive parameter names for clarity, such as 'name' instead of 'b'.